src/test/HostActivationTests/DotNetBuilder.cs

```diff
@@ -20,6 +21,18 @@ public class DotNetBuilder
         private readonly string _path;
         private readonly RepoDirectoriesProvider _repoDirectories;
 
+        public enum MockType
+        {
+            HostPolicy,
```

Comment:
When we add new hostpolicy apis for 3.0, we will need a hostpolicy2 mock for testing.